docs/source/reference/api-server/helm-values-spec.rst

```diff
@@ -923,6 +924,20 @@ Default: ``null``
       enabled: true
 
 
+.. _helm-values-auth-disableAnonymousAccess:
+
+``auth.disableAnonymousAccess``
```

Comment:
Can we make this consistent with other authentication scheme? e.g.

```yaml
auth:
  anonymous:
    enabled: false
```

This can also be more extensible since we may add:

```
auth:
  anonymouse:
     defaultRole: xxx
```

In the future.

Reply:
Updated